public/css/index.css

```diff
@@ -0,0 +1,118 @@
+:root {
+  /* colors */
+  --main-bg-color: #1e1e25;
+  --form-background-color: #27293d;
+  --form-input-main-border-color: #2a324e;
+  --form-input-focus-color: #fb00ff;
+  --form-input-focus-bg-color: #1d253b;
+  --button-main-color: gray;
+  --button-blue-color: #305cf4;
+  --button-pink-color: #d54fd7;
```

Comment:
co do nazewnictwa kolorów, jest wiele sposobów ale tu bym bardziej atomowe nazwy zostawił:
(ja tak preferuję), jest to bardziej skalowalne podejście na przyszłość
```css
:root {
  -c-orange: #F060D6;
  -c-red: #BB532E;
  -c-blue: #4C9FEB;

  -c-gray-1: #eee;
  -c-gray-2: #ccc;
  -c-gray-3: #555;
}
```

Reply:
https://css-tricks.com/what-do-you-name-color-variables/

Reply:
albo sposób z primary, primary-light, secondary, secondary-dark etc.